webextensions/api/permissions.json

```
@@ -6,7 +6,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/permissions",
           "support": {
             "chrome": {
-              "version_added": "≤59"
+              "version_added": "17"
```

:x: https://github.com/chromium/chromium/commit/fdacba32736298fa743c670d8146c736d41394b1 landed in [16.0.909.0](https://github.com/chromium/chromium/releases/tag/16.0.909.0) before the 16.0.912 release.

```suggestion
              "version_added": "16"
```